theano/gof/opt.py

```diff
@@ -1770,6 +1771,8 @@ def process_node(self, fgraph, node, lopt=None):
                 raise
         if replacements is False or replacements is None:
             return False
+        if replacements is True:
+            return True
```

Comment:
You mean that a `LocalOptimizer` could change the graph directly, without going through `replace_all_validate` and all that? That does not seem correct to me. And I don't think the comments of a PR is the right place to discuss such a change.